src/sections/shared/pagination/PaginationResultsInfo.tsx

```diff
@@ -18,25 +18,27 @@ export function PaginationResultsInfo({ paginationInfo, accumulated }: Paginatio
       accumulated === 1
         ? 'accumulated.one'
         : accumulated < paginationInfo.pageSize
-        ? 'accumulated.lessThanPageSize'
-        : 'accumulated.moreThanPageSize',
+          ? 'accumulated.lessThanPageSize'
+          : 'accumulated.moreThanPageSize',
     [accumulated, paginationInfo.pageSize]
   )
 
   return (
     <span className={styles.results}>
       {typeof accumulated === 'number'
         ? t(defineLocale(accumulated), {
-            accumulated: accumulated,
-            count: paginationInfo.totalItems,
-            item: paginationInfo.itemName
-          })
+          accumulated: accumulated,
+          count: paginationInfo.totalItems,
+          countFormat: new Intl.NumberFormat().format(paginationInfo.totalItems),
+          item: paginationInfo.itemName
+        })
         : t('results', {
-            start: paginationInfo.pageStartItem,
-            end: paginationInfo.pageEndItem,
-            item: paginationInfo.itemName,
-            count: paginationInfo.totalItems
-          })}
+          start: paginationInfo.pageStartItem,
+          end: paginationInfo.pageEndItem,
+          item: paginationInfo.itemName,
+          count: paginationInfo.totalItems,
+          countFormat: new Intl.NumberFormat().format(paginationInfo.totalItems)
```

Comment:
We are formatting `totalItems` twice, perhaps it would be cleaner to store this `formattedCount` in a variable, outside of the component's return function.
We could also use a `useMemo` hook to recalculate that value only if `totalItems` changes.
What do you think ?

Reply:
This is a great idea, I agree.